snap/implicit.go

```diff
@@ -29,13 +29,13 @@ var implicitSlots = []string{
 	"network-bind",
 	"network-control",
 	"network-observe",
-	"snap-control",
+	"snapd-control",
 	"system-observe",
 	"timeserver-control",
 	"timezone-control",
 	// XXX: those two should perhaps not be added by default
```

Comment:
Please drop the comment. We don't want people to have to manually say their desktop snaps should actually work.